packages/rancher-logging/generated-changes/overlay/app-readme.md

```diff
@@ -20,3 +20,7 @@ To collect logs from an entire cluster, users create cluster flows and cluster o
 - [Output](https://banzaicloud.com/docs/one-eye/logging-operator/crds/v1beta1/output_types/) - An output is a CRD (`Output`) that defines how to connect to logging providers so logs can be sent to the provider.
 
 For more information on how to configure the Helm chart, refer to the Helm README.
+
+## Systemd Configuration
+
+Some kubernetes distributions log to journald.  In order to collect these logs the `systemdLogPath` needs to be defined.  By default the `/run/log/journal` directory is used, some Linux distributes do not default to this path. For example Ubuntu defaults to `/var/log/journal`.  To determine your `systemdLogPath` run `cat /etc/systemd/journald.conf | grep -E ^\#?Storage | cut -d"=" -f2` on one of your nodes.  If `persistent` is returned your `systemdLogPath` should be `/var/log/journal`.  If `volatile` is returned  `systemdLogPath` should be `/run/log/journal`.  If `auto` is returned run `[ -d /var/log/journal ] && echo true` to check if `/var/log/journal` exists. If it does `systemdLogPath` should be `/var/log/journal` otherwise use `/run/log/journal`.  If any value not described here is returned rancher logging will not be able to collect control plane logs, set `Storage=volatile` in `/etc/systemd/journald.conf` and reboot your machine.
```

Comment:
1. `distributions` instead of `distributes`.
2. _By default the `/run/log/journal` directory is used, some Linux distributes do not default to this path._ - major nit, but comma doesn't seem to be applicable here. Either have it as separate sentences or maybe have it as _While by default the `/run/log/journal` directory is used, some Linux distributions do not default to this path._ which ties nicely to next "for example" sentence.
3. `rancher` should be changed to `Rancher`.
4. In the last sentence comma seems to be not the best choice either. Suggested wording: _If any value not described here is returned rancher logging will not be able to collect control plane logs. To address this issue set `Storage=volatile` in `/etc/systemd/journald.conf`, reboot your machine and set `systemdLogPath` per the instructions above._

Reply:
I was struggling with how to word #4 thank you